Procfile

```diff
@@ -1,2 +1,2 @@
-web: env RAILS_ENV=$RAILS_ENV env PORT=$PORT bundle exec unicorn -E $RAILS_ENV -c config/unicorn.rb
-jobs: env RAILS_ENV=$RAILS_ENV bundle exec bin/delayed_job run
```

Comment:
How does the `delayed_job` figure out which `RAILS_ENV` to run in?  Or is that handled by the `.env` file?